components/brave_player/core/browser/brave_player_service.h

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#ifndef BRAVE_COMPONENTS_BRAVE_PLAYER_CORE_BROWSER_BRAVE_PLAYER_SERVICE_H_
+#define BRAVE_COMPONENTS_BRAVE_PLAYER_CORE_BROWSER_BRAVE_PLAYER_SERVICE_H_
+
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "base/component_export.h"
+#include "base/files/file_path.h"
+#include "base/functional/callback.h"
+#include "base/memory/raw_ptr.h"
+#include "base/memory/weak_ptr.h"
+#include "components/keyed_service/core/keyed_service.h"
+
+class GURL;
+
+namespace content {
+class WebContents;
+}  // namespace content
```

Comment:
I believe you seem to have polluted `core` with `content` requirements

Reply:
Thanks, it seems I should move the delegation to the tab helper.